mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -155,6 +162,10 @@ func IsMesheryRunning(currPlatform string) (bool, error) {
 		}
 	}
 
+	if err != nil {
+		return false, errors.Wrap(err, "Meshery not running locally")
+	}
+
```

Comment:
No need to add this thing, we are returning False status in the next line.

Reply:
Agreed but that Golang rules... 🙂 Once declared, the variable has to be used right?